projects/plugins/jetpack/extensions/blocks/donations/edit.js

```diff
@@ -61,9 +60,8 @@ const Edit = props => {
 			return;
 		}
 
-		// Set fake products when plan should be upgraded or there is no connection to Stripe so users can still try the
-		// block in the editor.
-		if ( result.should_upgrade_to_access_memberships || result.connect_url ) {
```

Comment:
`should_upgrade_to_access_memberships` is currently expected to passed via the API to the Donations block. As such I chose not to remove the param from the API in D108873-code until this code is deployed on WPCOM and released in a new JP version.

Reply:
I believe we should remove all `should_upgrade_to_access_memberships` in this PR.

> However, in JavaScript result.should_upgrade_to_access_memberships would return undefined if should_upgrade_to_access_memberships is not passed which is okay in this case. 

That is actually great news as it will make the feature reto-compatible once D108873-code is out!